### PR TITLE
Add first phase of Personalization Profile

### PR DESCRIPTION
**Pull request by ncksllvn**

### CODE REVIEW

File: src/js/user-profile/components/ProfileAccordion.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        </AccordionTab>
+        <AccordionTab onClick={this.onAccordionTabClick} activeAccordionId={this.state.activeAccordionId} id="insurance-information" title="Insurance Information">
+          <h3>Insurance Information</h3>
+        </AccordionTab>
+      </Accordion>
+    );
+  }
+}
+
+export default ProfileAccordion;
```
Comment (ncksllvn) [inline src/js/user-profile/components/ProfileAccordion.jsx]: This file isn't used currently but I didn't want to just delete it.

File: src/js/user-profile/actions/index.js [hunk 2]
```diff
@@ -97,3 +97,71 @@ export function removeSavedForm(formId) {
...
+          stateCode: 'KY',
+          zipCode: '41042',
+          countryName: 'USA'
+        }
+      ],
+      toursOfDuty: [
+        {
+          serviceBranch: 'Navy',
+          rank: 'First Lieutenant',
+          dateRange: {
```
Comment (ncksllvn) [inline src/js/user-profile/actions/index.js]: I forgot we'll also need the rank.

File: src/js/user-profile/actions/index.js [hunk 3]
```diff
@@ -97,3 +97,71 @@ export function removeSavedForm(formId) {
       .catch(() => dispatch(removingSavedFormFailure()));
   };
 }
+
+export function fetchExtendedProfile() {
```
Comment (bshyong) [inline src/js/user-profile/actions/index.js]: FYI we will be breaking this out into multiple actions to "progressively enhance" the profile object based on the endpoints available, e.g. address, military history, etc.

**Comment by bshyong** [comment]
LGTM; merging into our `personalization-dev` branch

**Pull request merged**